lib/faraday/http_cache.rb

```diff
@@ -302,6 +324,29 @@ def log_request
       @logger.debug(line)
     end
 
+    # Internal: instruments the request processing.
+    #
+    # Returns nothing.
+    def instrument(env)
+      return if @instrumenter.nil?
+
+      payload = {
+        env: env,
```

Comment:
It looks like no test checks the env on the instrumented payload, do we really need to pass it in? If so, maybe a test should explicitly fail in case it is removed.